consistent-sampling/build.gradle.kts

```diff
@@ -4,6 +4,7 @@ plugins {
 }
 
 description = "Sampler and exporter implementations for consistent sampling"
+otelJava.moduleName.set("io.opentelemetry.contrib.samplers")
```

Comment:
looks like we are using this package in two modules

Reply:
I've made the following changes:
- `:samplers` base package changed to `io.opentelemetry.contrib.sampler`
- `:consistent-sampling` base package changed to `io.opentelemetry.contrib.sampler.consistent`